install/k8s/README.md

```diff
@@ -0,0 +1,61 @@
+# Kubernetes Setup for Contiv
+
+This document details the setup instructions for Kubernetes version 1.4 and higher for CentOS 7
+
+### Setup Kubernetes Cluster
+Install kubernetes 1.4 or higher using http://kubernetes.io/docs/getting-started-guides/kubeadm/
+
+(OR) Alternatively do the following:
```

Comment:
Gaurav wanted to have a simple script to install k8s for testing. I am moving the K8s install to a devguide and also cleaning up the scripts to be simple for dev only usecases.